test/phonons.jl

```diff
@@ -105,4 +105,77 @@ end
     end
 end
 
+# Return the torus for x ∈ [-½, ½[ instead of [0, 1[.
```

Comment:
looks like a test script rather than a test that should be in DFTK, don't commit it yet and wait until the proper phonon PR

Reply:
When you implement this as fourier space purely, you can test it matches the G_to_r/r_to_G approach